File: Examenes/EX1/22-1/JAVA/EduSoftLP2Model/src/pe/edu/pucp/edusoftlp2/model/Curso.java
package pe.edu.pucp.edusoftlp2.model;
import java.util.ArrayList;
public class Curso {
    private int idCurso;
    private Especialidad especialidad;
    private Modalidad modalidad;
    private ArrayList<Docente> docentes;
    private String nombre;
    private String clave;
    private String semestre;
    private boolean reqComputadora;
    private boolean reqDispMovil;
    private boolean reqDispEspecial;
    private String sumilla;
    private byte[] foto;
    private byte[] silabo;

    public int getIdCurso() {
        return idCurso;
    }

    public void setIdCurso(int idCurso) {
        this.idCurso = idCurso;
    }

    public Especialidad getEspecialidad() {
        return especialidad;
    }

    public void setEspecialidad(Especialidad especialidad) {
        this.especialidad = especialidad;
    }

    public Modalidad getModalidad() {
        return modalidad;
    }

    public void setModalidad(Modalidad modalidad) {
        this.modalidad = modalidad;
    }

    public ArrayList<Docente> getDocentes() {
        return docentes;
    }

    public void setDocentes(ArrayList<Docente> docentes) {
        this.docentes = docentes;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getClave() {
        return clave;
    }

    public void setClave(String clave) {
        this.clave = clave;
    }

    public String getSemestre() {
        return semestre;
    }

    public void setSemestre(String semestre) {
        this.semestre = semestre;
    }

    public boolean isReqComputadora() {
        return reqComputadora;
    }

    public void setReqComputadora(boolean reqComputadora) {
        this.reqComputadora = reqComputadora;
    }

    public boolean isReqDispMovil() {
        return reqDispMovil;
    }

    public void setReqDispMovil(boolean reqDispMovil) {
        this.reqDispMovil = reqDispMovil;
    }

    public boolean isReqDispEspecial() {
        return reqDispEspecial;
    }

    public void setReqDispEspecial(boolean reqDispEspecial) {
        this.reqDispEspecial = reqDispEspecial;
    }

    public String getSumilla() {
        return sumilla;
    }

    public void setSumilla(String sumilla) {
        this.sumilla = sumilla;
    }

    public byte[] getFoto() {
        return foto;
    }

    public void setFoto(byte[] foto) {
        this.foto = foto;
    }

    public byte[] getSilabo() {
        return silabo;
    }

    public void setSilabo(byte[] silabo) {
        this.silabo = silabo;
    }
    
}

File: Examenes/EX1/22-1/CSharp/EduSoftLP2/EduSoftLP2Model/Curso.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EduSoftLP2Model
{
    public class Curso
    {
        private int _idCurso;
        private Especialidad _especialidad;
        private Modalidad _modalidad;
        private BindingList<Docente> _docentes;
        private string _nombre;
        private string _clave;
        private string _semestre;
        private bool _reqComputadora;
        private bool _reqDispMovil;
        private bool _reqDispEspecial;
        private string _sumilla;
        private byte[] _foto;
        private byte[] _silabo;
        public int IdCurso { get => _idCurso; set => _idCurso = value; }
        public Especialidad Especialidad { get => _especialidad; set => _especialidad = value; }
        public Modalidad Modalidad { get => _modalidad; set => _modalidad = value; }
        public BindingList<Docente> Docentes { get => _docentes; set => _docentes = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public string Clave { get => _clave; set => _clave = value; }
        public string Semestre { get => _semestre; set => _semestre = value; }
        public bool ReqComputadora { get => _reqComputadora; set => _reqComputadora = value; }
        public bool ReqDispMovil { get => _reqDispMovil; set => _reqDispMovil = value; }
        public bool ReqDispEspecial { get => _reqDispEspecial; set => _reqDispEspecial = value; }
        public string Sumilla { get => _sumilla; set => _sumilla = value; }
        public byte[] Foto { get => _foto; set => _foto = value; }
        public byte[] Silabo { get => _silabo; set => _silabo = value; }
    }
}
